templates/docs/examples/base/code-inline-dark.html

```diff
@@ -0,0 +1,14 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code - inline / dark{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
+<div class="p-strip--dark">
```

Comment:
The standalone example for dark code doesn't show dark strip (because it only contains base styles).

And while in case of components we would just add any other needed component to standalone CSS, we don't want to do this in case of base examples (because we want them to work with just the base).

So I guess the easiest way to workaround it would be to add inline background here (to make sure background is dark even without strip CSS):

```suggestion
<div class="p-strip--dark" style="background: #111">
```